control_plane/src/storage.rs

```diff
@@ -199,23 +199,42 @@ impl PageServerNode {
         bail!("pageserver failed to start in {} seconds", RETRIES);
     }
 
-    pub fn stop(&self) -> anyhow::Result<()> {
+    pub fn stop(&self, immediate: bool) -> anyhow::Result<()> {
         let pid = read_pidfile(&self.pid_file())?;
         let pid = Pid::from_raw(pid);
-        if kill(pid, Signal::SIGTERM).is_err() {
-            bail!("Failed to kill pageserver with pid {}", pid);
+        if immediate {
+            println!("Stop pageserver immediately");
```

Comment:
> `println!`

Out of curiosity, why this and not use `log::something!` instead? 
You seem to be doing logs of i/o here anyway (reading from the FS, ordering the process to be killed), so presumably logging is also fine?